vega_sim/scenario/ideal_market_maker_v2/agents.py

```diff
@@ -235,35 +264,40 @@ def step(self, vega_state: VegaState):
 
         buy_order, sell_order = None, None
 
-        for order in (
-            vega_state.market_state[self.market_id]
-            .orders[self.vega.wallet.public_key(self.wallet_name)]
-            .values()
-        ):
-            if order.side == vega_protos.SIDE_BUY:
-                buy_order = order
-            else:
-                sell_order = order
-
-        # self.vega.submit_simple_liquidity(
-        #     wallet_name=self.wallet_name,
-        #     market_id=self.market_id,
-        #     commitment_amount=self.commitment_amount,
-        #     fee=0.002,
-        #     reference_buy="PEGGED_REFERENCE_MID",
-        #     reference_sell="PEGGED_REFERENCE_MID",
-        #     delta_buy=self.bid_depth,
-        #     delta_sell=self.ask_depth,
-        # )
-
-        self._place_orders(
-            buy_offset=self.bid_depth,
-            sell_offset=self.ask_depth,
-            volume=1,
-            buy_order=buy_order,
-            sell_order=sell_order,
+        try:
+            for order in (
+                vega_state.market_state[self.market_id]
+                .orders[self.vega.wallet.public_key(self.wallet_name)]
+                .values()
+            ):
+                if order.side == vega_protos.SIDE_BUY:
+                    buy_order = order
+                else:
+                    sell_order = order
+            
+        except Exception as error:
+            print(error)
+            pass
```

Comment:
I don't think this should be throwing any errors? What's getting thrown in here (And why do we want to ignore whatever error *is* thrown?)

As a side note: If we were just logging errors we'd want to use `logger.exception` rather than print, but I suspect we don't want the try/except here at all.

Reply:
It got a 'Key error' when searching the orders, but I think the issue would be fixed after the latest change, will delete this.